alfred-telemetry-solr/README.md

```diff
@@ -80,23 +80,23 @@ At the moment following metrics are included, for each core tracked by solr:
 
 | Name                                                                   |
 | -----------------------------------------------------------------------|
-| alfresco_nodes{core=<core>,feature="Alfresco Nodes in Index"}          |
-| alfresco_nodes{core=<core>,feature="Alfresco Transactions in Index"}   |
-| alfresco_nodes{core=<core>,feature="Alfresco Stated in Index"}         |
-| alfresco_nodes{core=<core>,feature="Alfresco Unindexed Nodes"}         |
-| alfresco_nodes{core=<core>,feature="Alfresco Error Nodes in Index"}    |
-| alfresco_acls{core=<core>,feature="Alfresco Acls in Index"}            |
-| alfresco_acls{core=<core>,feature="Alfresco Acl Transactions in Index"}|
+| alfresco.nodes{core=<core>,feature="Alfresco Nodes in Index"}          |
+| alfresco.nodes{core=<core>,feature="Alfresco Transactions in Index"}   |
+| alfresco.nodes{core=<core>,feature="Alfresco Stated in Index"}         |
+| alfresco.nodes{core=<core>,feature="Alfresco Unindexed Nodes"}         |
+| alfresco.nodes{core=<core>,feature="Alfresco Error Nodes in Index"}    |
+| alfresco.acls{core=<core>,feature="Alfresco Acls in Index"}            |
```

Comment:
but I don't think `acls in index`  + `acl transactions in index` can semantically be added together ?